### DISCUSSION TITLE
Bookmarks

### DISCUSSION THREAD

**Discussion by munen** [200ok-ch/organice#760] [Q&A] [answered]
@tarnung, I love the functionality you just introduced with [bookmarks](https://github.com/200ok-ch/organice/pull/758).

What do you think about tweaking the workflow slightly to allow for deleting a bookmark? I believe that users will create bookmarks just for testing, and already they have 'junk' data (I did that at least^^). So, when a user has entered a search string which already is a bookmark (maybe by selecting the bookmark), the ⭐ action could change to a :wastebasket: icon to delete it, again?

**Comment by tarnung** [comment, marked as answer]
I wanted to keep the UI surface as small as possible. Your idea is in line with this. I like it. Expect a pull request soon-ish.

**Reply by munen** [reply]
Nice :rocket: :pray:

**Comment by munen** [comment]
Mr. @tarnung is the Master of speed. The proposal is already implemented and merged: https://github.com/200ok-ch/organice/pull/761

:100:
